pages/_app.tsx

```diff
@@ -1,4 +1,4 @@
-import CssBaseline from '@material-ui/core/CssBaseline';
+import CssBaseline from '@mui/material/CssBaseline';
```

Comment:
This may be the reason the general font size seems to have changed from 14px to 16px in multiple places.

https://mui.com/material-ui/react-css-baseline/

> Typography
> No base font-size is declared on the <html>, but 16px is assumed (the browser default). You can learn more about the implications of changing the <html> default font size in [the theme documentation](https://mui.com/material-ui/customization/typography/#typography-html-font-size) page.

Reply:
Indeed it was.
https://mui.com/material-ui/guides/migration-v4/#cssbaseline

https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1467/commits/899022f498edbef1e2c102fc2fae950fb381618a